test/unit/jest.config.json

```diff
@@ -5,8 +5,9 @@
 		"packages/**/*.js"
 	],
 	"moduleNameMapper": {
-		"@wordpress\\/(blocks|components|editor|utils|edit-post|viewport|core-data|core-blocks|nux)$": "$1",
-		"@wordpress\\/(api-request|blob|core-data|data|date|dom|deprecated|element|plugins|postcss-themes)$": "packages/$1/src"
+		"@wordpress\\/(blocks|components|editor|utils|edit-post|viewport|core-blocks|nux)$": "$1",
+		"@wordpress\\/(browserslist-config)$": "packages/$1",
+		"@wordpress\\/(.*)$": "packages/$1/src"
```

Comment:
@aduth @youknowriad - do we want to run our tests against `src`, `build` or `build-module`? What if we provide `src` locally and `build-module` on Travis?

Reply:
I think we should only run `src` for both.

Reply:
We are all good then 👍